Exercise 1: Opioids Exercise

radamson opened the pull request:
WIP Opioids Notebook

See the notebook here: https://github.com/mitre/fhir-exercises/blob/opioids-1/Opioids%20Use%20Case%20Example.ipynb

jawalonoski reviewed (commented):
- Add Learning Objectives and Key Concepts before Introduction.
- "reasonable return" should be "reasonably return"
- Remove or resolve TODO: regarding dynamic medication name

Overall, looks good.

radamson commented:
> - [x] Add Learning Objectives and Key Concepts before Introduction.
> - [x] "reasonable return" should be "reasonably return"
> - [x] Remove or resolve TODO: regarding dynamic medication name

Addressed the comments. Instead of making the medication name dynamic, I added a list comprehension to check for a tramadol MedicationRequest since I think making it dynamic might make the rest of the narrative harder, especially if the first medication returned ends up not being an opioid

radamson commented:
Made a few small additions. The main things:

Added an `Icons in this Guide` section:
![Screen Shot 2022-01-27 at 3 13 02 PM](https://user-images.githubusercontent.com/41651655/151436359-486eed0e-b4af-446f-a9fe-2dde5566b890.png)

Added a very basic FHIR Terminology API call (`$lookup`) and links to further documentation.
![Screen Shot 2022-01-27 at 3 14 25 PM](https://user-images.githubusercontent.com/41651655/151436530-295fa0c5-becf-4da9-9d85-d22299d9d1f4.png)

radamson commented:
Added `DataFrame.describe` to align with `skim` used in the R notebook

![Screen Shot 2022-01-27 at 3 58 15 PM](https://user-images.githubusercontent.com/41651655/151442683-647e1535-b056-4bfc-a8f4-827d424d58dd.png)

radamson commented:
https://github.com/mitre/fhir-exercises/pull/2/commits/6650f26bc71d272fa55efa3f0f2e8462a0eec931  - Replaced Pie Charts with Tables to better align with R notebook

![Screen Shot 2022-01-27 at 5 19 13 PM](https://user-images.githubusercontent.com/41651655/151453289-2d4aa01d-7727-494a-b610-211a3d267d1c.png)
.

radamson commented:
https://github.com/mitre/fhir-exercises/pull/2/commits/23bef06914464d69c39c912a0817dca773e2a03a - More updates based on the R notebook.

Defined `get_atc_class` method and applied it to all rows.

![Screen Shot 2022-01-27 at 5 52 15 PM](https://user-images.githubusercontent.com/41651655/151456689-40100426-0d2c-4ea5-8cd9-06eacbe91290.png)

jawalonoski reviewed (approved):
The commit I just made fixes some minor typos in the narrative, and adds a short recap at the end that essentially repeats the learning objectives to close out the notebook.

dehall commented on README.md:
````diff
@@ -2,7 +2,45 @@
+jupyter-lab
+```
+
+By default, this should open Jupyter lab at 
````
```suggestion
By default, this should open Jupyter lab at http://localhost:8888/lab
```
